app/views/layouts/application.html.slim

```diff
@@ -18,28 +18,29 @@ html lang="#{I18n.locale || I18n.default_locale}" data-default-locale="#{I18n.de
     meta name='current-user' content=current_user&.to_page_context&.to_json
   body.d-flex.flex-column.h-100
     #content
-      nav.navbar.bg-light.navbar-expand-lg.mb-0 role='navigation' data-bs-theme="light"
-        .container
-          = link_to home_index_path, class: "navbar-brand"
-            i.fa-solid.fa-anchor.me-2#logo
-            = application_name
-          button.navbar-toggler data-bs-target='#navbar-collapse' data-bs-toggle='collapse' type='button' aria-expanded='false' aria-label='Toggle navigation' aria-controls='navbar-collapse'
-            span.navbar-toggler-icon
-          #navbar-collapse.collapse.navbar-collapse
-            = render('navigation')
-            ul.nav.navbar-nav.ms-auto
-              = render('session')
+      header.py-3
+        nav.navbar.bg-light.navbar-expand-lg.fixed-top.text-bg-light.fs-6-p-0 role='navigation'
+          .container
+            = link_to home_index_path, class: "navbar-brand"
+              i.fa-solid.fa-anchor.me-2.text-secondary
+              = application_name
```

Comment:
Hovering over the application name doesn't quite fit into the remaining hover experience of the nav bar, does it?

<img width="334" alt="Bildschirmfoto 2023-11-09 um 18 02 24" src="https://github.com/openHPI/codeharbor/assets/7300329/2266b2b1-fd7d-4f42-88fb-cd20fb141cfa">